web_viewer/index.html

```diff
@@ -326,7 +326,27 @@
 
         console.debug("Wasm loaded. Starting app…");
 
-        let handle = new wasm_bindgen.WebHandle();
+        const query = new URLSearchParams(window.location.search);
+        /*
+          url: Option<String>,
+          manifest_url: Option<String>,
+          render_backend: Option<String>,
+          hide_welcome_screen: Option<bool>,
+          panel_state_overrides: Option<PanelStateOverrides>,
+          fullscreen: Option<FullscreenOptions>,
+        */
+        const options = {
+          app_id: query.get("app_id"),
+          url: query.get("url"),
+          manifest_url: query.get("manifest_url"),
+          render_backend: query.get("renderer"),
+          hide_welcome_screen: query.has("hide_welcome_screen"),
+
+          notebook: get_query_bool(query, "notebook", false),
+          persist: get_query_bool(query, "persist", true),
```

Comment:
It's a bit annoying that the content and order of the comment does not match the code